### ISSUE TITLE
2.0-jessie image got critical vulnerable during docker hub security scan

### ISSUE THREAD

**Issue by natemcmaster** [dotnet/dotnet-docker#353]
_From @wsun4ipipeline on December 14, 2017 20:7_

## Steps to reproduce the issue

1.push the docker image microsoft/aspnetcore:2.0.4-jessie to hub.docker
2.check the tag tap for  security scan result after couple hours when the result available

## Expected behavior
no critical vulnerable

## Actual behavior
20 of 239 components are vulnerable

## Output of  docker hub security scan

```
ADD file:1dd78a123212...c3c92b63b73d8d1 in / 
Compressed size: 50.2MB

13 vulnerable components 


/bin/sh -c apt-get up... /var/lib/apt/lists/*
Compressed size: 17.7MB
 
6 vulnerable components 

/bin/sh -c for versio...tore.tar.gz; done
Compressed size: 32.8MB

1 vulnerable component
```

_Copied from original issue: aspnet/aspnet-docker#344_

**Comment by natemcmaster** [comment]
cc @MichaelSimons ideas?

**Comment by natemcmaster** [comment]
_From @wsun4ipipeline on December 15, 2017 13:54_

and I tried scan 2.0.4-stretch too, it also has 27 vulnerable.
`ADD file:eb2519421c97...4d2ddb7de69e52a in / Compressed size: 43.0MB
13 vulnerable components

/bin/sh -c apt-get up... /var/lib/apt/lists/*Compressed size: 21.8MB
13 vulnerable components


/bin/sh -c for versio...tore.tar.gz; doneCompressed size: 32.8MB
1 vulnerable component
`

if those unix componets is not used by asp.net core framework, Could you build  a 'clean version ' by removing them, and let the end user installed in their docker file if they need them.

**Comment by natemcmaster** [comment]
_From @MichaelSimons on December 15, 2017 14:48_

@wsun4ipipeline - would you move this issue to the [dotnet/dotnet-docker repo](https://github.com/dotnet/dotnet-docker)?  The vulnerabilities reside in the base layers of the aspnetcore images - e.g. microsoft/dotnet and the base OS.

**Comment by wsun4ipipeline** [comment]
Just a update for Info:  This issue cause our IT security team  denied our deploy plan using docker for our new app. And our team have to  deploy our new app to ubuntu server directly, so our IT team can easily keep the server updated..

**Comment by MichaelSimons** [comment]
@wsun4ipipeline - Have you considered creating an Ubuntu based image yourself to run your app in and would that satisfy your IT team's requirements.  You can easily create an Ubuntu based Dockerfile that installs the .NET Core runtime via apt-get - https://www.microsoft.com/net/download/linux

**Comment by richlander** [comment]
We resolved this in a couple ways in 2.1:

- Added Ubuntu 18.04/Bionic
- Added Alpine
- Dropped Jessie

We believe that these choices are the best avenues available to us.
